### DISCUSSION TITLE
My education application is going rejected asking for upload photo using device camera still i am using device camera can anyone help me

### DISCUSSION THREAD

**Discussion by vikukumar** [community/community#136758] [GitHub Education]
### Hi everyone,

My education application is going rejected asking for upload photo using device camera still i am using device camera can anyone help me

**Comment by Deekshit7483** [comment]
Hello,

where it's getting reject.. Are you trying to publish the application??

**Comment by Carlos-Levi** [comment]
Hi Vikukumar,

I understand how frustrating it can be to have your education application rejected despite following the instructions. Here are a few tips that might help:

 Check the Photo Quality: Ensure that the photo you’re taking with your device camera is clear, well-lit, and free from any reflections or shadows. The text on your document should be fully legible.

Use a Different Device or Camera: Sometimes, the camera on one device might not capture the image clearly enough. If possible, try using a different device or camera to take the photo.

 Avoid Using Filters or Effects: Make sure the camera app you’re using doesn’t apply any filters or effects that could alter the appearance of the document.

File Format and Size: Double-check the file format and size of the image you’re uploading. GitHub may have specific requirements for the type and size of the file. If you’re submitting a document, see if they accept a PDF format, which can sometimes be easier to read and process.

 Contact GitHub Support: If you’ve tried these suggestions and are still facing issues, consider reaching out to GitHub Support for further assistance. Explain that you’re using your device camera as instructed but still facing rejections.

I hope one of these tips helps resolve your issue. Good luck with your application!
